packages/mcp-auth/src/utils/transpile-protected-resource-metadata.ts

```diff
@@ -0,0 +1,9 @@
+import { type CamelCaseProtectedResourceMetadata } from '../types/oauth.js';
+import { type ProtectedResourceConfig } from '../types/protected-resource.js';
+
+export const transpileProtectedResourceMetadata = (
+  metadata: ProtectedResourceConfig['metadata']
+): CamelCaseProtectedResourceMetadata => ({
+  ...metadata,
+  authorizationServers: metadata.authorizationServers?.map((server) => server.metadata.issuer),
+});
```

Comment:
i'm confused here. it seems `authorizationServers` already contains full auth server data, so what does `jwksUri` in `ProtectedResourceMetadata` do?

Reply:
The `jwksUri` in protected resource metadata points is used for verifying the resource's response signatures (`signed_metadata`). This is different from the authorization server's jwksUri which is used for token validation.